network_wrangler/models/gtfs/gtfs.py

```diff
@@ -52,21 +109,99 @@ class GtfsModel(DBModelMixin):
     }
 
     table_names: ClassVar[list[str]] = [
-        "frequencies",
+        "agency",
         "routes",
         "shapes",
         "stops",
         "trips",
         "stop_times",
     ]
 
-    optional_table_names: ClassVar[list[str]] = ["agencies"]
+    optional_table_names: ClassVar[list[str]] = ["frequencies"]
 
     def __init__(self, **kwargs):
         """Initialize GTFS model."""
         self.initialize_tables(**kwargs)
 
         # Set extra provided attributes.
         extra_attr = {k: v for k, v in kwargs.items() if k not in self.table_names}
-        for k, v in extra_attr:
+        for k, v in extra_attr.items():
             self.__setattr__(k, v)
+
+    def __repr__(self) -> str:  # noqa: PLR0912
+        """Return a string representation of the GtfsModel with table summaries."""
+        lines = ["GtfsModel:"]
+
+        # Add agency info if available
```

Comment:
I would encapsulate all of these in a directly callable method e.g. `summary()` and then have `__repr__()` call that.